### PR TITLE
[REF] mail: `rtcCallViewer`, review and simplify scss

### PR DESCRIPTION
**Pull request by xlu-odoo**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2835842

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by xlu-odoo** [comment]
@Brieuc-brd could you look this one? 🌞

**Review by Brieuc-brd — COMMENTED** [review]
@xlu-odoo 👋 
Some remarks below 👇 👍

**Comment by xlu-odoo** [comment]
@Brieuc-brd 👀

File: addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml [hunk 1]
```diff
@@ -3,25 +3,26 @@
...
                 </t>
                 <t t-if="rtcCallViewer.layout !== 'spotlight'">
                     <div
-                        class="o_RtcCallViewer_grid"
+                        class="o_RtcCallViewer_grid d-flex align-items-center h-100"
                         t-att-class="{
-                            'o-isTiled': rtcCallViewer.layout === 'tiled',
-                            'o-isSidebar': rtcCallViewer.layout === 'sidebar',
+                            'o-isTiled justify-content-center w-100': rtcCallViewer.layout === 'tiled',
+                            'flex-wrap': rtcCallViewer.layout !== 'tiled',
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml:24]: I think there's an issue here. According to the css you removed at lines 43 to 59, 
- `flex-wrap` should be triggered with `o-isTiled`.
- `flex-column` should be triggered in all cases except when `o-isTiled` is triggered.

**Review by Brieuc-brd — COMMENTED** [review]
@xlu-odoo 👋 
Small remark below 😉

**Comment by xlu-odoo** [comment]
@Brieuc-brd updated

**Comment by xlu-odoo** [comment]
@Brieuc-brd

**Review by Brieuc-brd — COMMENTED** [review]
@xlu-odoo 👋 
Just a last remark, otherwise LGTM 👍

File: addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml [hunk 2]
```diff
@@ -3,24 +3,25 @@
 
 <t t-name="mail.RtcCallViewer" owl="1">
     <t t-if="rtcCallViewer">
-        <div class="o_RtcCallViewer" t-att-class="{ 'o-fullScreen': rtcCallViewer.isFullScreen, 'o-isMinimized': rtcCallViewer.isMinimized }" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_RtcCallViewer d-flex flex-column align-items-center justify-content-center bg-dark" t-att-class="{'o-fullScreen fixed-top vw-100 vh-100': rtcCallViewer.isFullScreen, 'o-isMinimized': rtcCallViewer.isMinimized, 'h-50': !rtcCallViewer.isMinimized, 'position-relative': !rtcCallViewer.isFullScreen }" t-attf-class="{{ className }}" t-ref="root">
```
[the remark this answers is not in the document]
Comment (xlu-odoo) [reply addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml]: Updated! Removing draft?

**Review by alexkuhn — APPROVED** [review]
Cool, I see you fixed the issue of tiny tile!
Thanks!

@robodoo r+

**Pull request closed without merging**